app/models/concerns/rad_user.rb

```diff
@@ -60,11 +60,7 @@ module RadUser
             "WHERE security_roles.#{permission} = TRUE)")
     }
 
-    scope :inactive, lambda {
-      joins(:user_status)
-        .where('user_statuses.active = FALSE OR (invitation_sent_at IS NOT NULL AND invitation_accepted_at IS NULL)')
```

Comment:
i'm not really sure what the ramifications are of this but I don't think we should combine the user status active logic with the devise authentication active logic